app/controllers/CharitiesController.scala

```diff
@@ -34,7 +33,7 @@ class CharitiesController @Inject()(servicesConfig: ServicesConfig,
                                     implicit val appConfig: AppConfig,
                                     authAction: AuthAction,
                                     mcc: MessagesControllerComponents)
-  extends FrontendController(mcc) with I18nSupport {
+  extends FrontendController(mcc)  {
```

Comment:
I'm not sure why this controller exists? It should be deleted, which should also help with the current test coverage.